hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/DeleteBucketHandler.java

```diff
@@ -143,6 +145,10 @@ private void deleteFSBucketRecursive(OzoneVolume vol, OzoneBucket bucket) {
       OzoneConfiguration clientConf = new OzoneConfiguration(getConf());
       clientConf.set(FS_DEFAULT_NAME_KEY, hostPrefix);
       FileSystem fs = FileSystem.get(clientConf);
+      if (handleSpecialVolumeTmp(path, (RootedOzoneFileSystem) fs,
+          clientConf)) {
```

Comment:
we should attempt deleting the bucket if it exists and if it doesn't do nothing. Here we are simply returning.

Reply:
In this case, if handleSpecialVolumeTmp() returns true that means we have encountered the hashed bucket (that would later throw bucket doesn't exist error). 

Do you suggest that we do not return here and just skip the fs.delete block (but let it proceed further)?